aperturedb/Utils.py

```diff
@@ -120,7 +120,7 @@ def __init__(self, connector: Connector, verbose=False):
         self.verbose = verbose
 
     def __repr__(self):
-        return self.status()
+        return f"{id(self)=}"
```

Comment:
We will need to bump min python to 3.8 to use the {var=}

Reply:
oh. we do not need the = . It's just a habbit. I will remove that